docs/en/operation/User-CLI.md

```diff
@@ -1101,18 +1101,52 @@ $ ./bin/alluxio fs leader
 ### load
 
 The `load` command moves data from the under storage system into Alluxio storage.
+For example, `load` can be used to prefetch data for analytics jobs.
```

Comment:
@jja725 i'm assuming it is appropriate to keep the docs here because the load feature was added in `dora` and so it's already in `main`